ember-app/package.json

```diff
@@ -21,6 +21,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.0",
+    "ember-browserify": "1.1.8",
```

Comment:
I don't object to using it, but I'd prefer to bring it in with a first example of actually using it for awesome. Otherwise we're just carrying around dead weight.